decidim-core/app/controllers/decidim/user_activities_controller.rb

```diff
@@ -19,7 +19,12 @@ def index
     private
 
     def user
-      @user ||= current_organization.users.find_by(nickname: params[:nickname])
+      return nil unless params[:nickname]
```

Comment:
Here return "nil" is possible but not required, prefer implicit return : 
```suggestion
      return unless params[:nickname]
```